add AI_POLICY.md (#164)
tc39/how-we-work: AI_POLICY.md

# Use of LLMs and similar tools

> Any contributions or comments must be your own writing, not the product of LLMs or other tools. Do not prompt an LLM to expand on or explain your idea and then post the output; just provide the idea itself.

Reviewer: ```suggestion
Any contributions or comments must be your own writing, not the product of LLMs ("large language models", or other AI/machine learning-based text generators) or other tools. Do not prompt an LLM to expand on or explain your idea and then post the output; just provide the idea itself.
```

Reviewer 2: >  other AI/machine learning-based text generators

is already covered by "or other tools" immediately following your suggestion

Reviewer: It's not intended to; I'm expanding the definition of LLM specifically into a full category. "Other tools" (presumably?) covers anything else that can generate text. But we could delete "or other tools" too.

Reviewer 2: We could just not mention the underlying technology.

```suggestion
Any contributions or comments must be your own writing, not the product of AI text generators. Do not prompt an AI text generator to expand on or explain your idea and then post the output; just provide the idea itself.
```

Reviewer 3: "a generative AI tool" might read better in that case.

Author: I feel it's important to mention LLMs because those are what people know about, and it may not occur to them that LLMs are "AI text generators".

I expanded the abbreviation but have otherwise left the text as-is, since I think it's clearer than any of these suggestions.

> Any contributions or comments must be your own writing, not the product of LLMs or other tools. Do not prompt an LLM to expand on or explain your idea and then post the output; just provide the idea itself.

Reviewer 4: I understand and respect the concern about reading AI-generated text. I'm not advocating for blindly pasting in ChatGPT output or relying on tools to generate ideas wholesale. But there's an important distinction between that and using an LLM to help refine or clarify your own thinking, especially when you fully understand and take responsibility for the result.

This policy defines how we work as a community. While individual preferences are valid, “I explicitly do not want to read the output of such tools” isn’t, by itself, a solid foundation for a shared policy. It shifts the focus from collective principles to personal taste.

Perhaps we can agree on a policy that prioritizes accountability and authorship, regardless of whether a tool was used along the way. For example:

> “Contributors may use tools, including LLMs, to assist with phrasing or clarity, provided they fully understand and take responsibility for the final content. Do not submit material you cannot explain or defend. Contributions should reflect your personal understanding, not generated material passed off as your own.”

This would discourage misuse without excluding contributors who responsibly use modern tools to express their ideas more clearly or accessibly.

> Any contributions or comments must be your own writing, not the product of LLMs or other tools. Do not prompt an LLM to expand on or explain your idea and then post the output; just provide the idea itself.

Reviewer: Agreed. There is a very big distinction between "text that you wrote, grammar-checked with a tool, and iterated on in discussion with others" and "text that you wrote some of, and generated the rest of with a text generator". Is there a theoretically reasonable path one can walk where all the generated text is equivalent to what one would have written by themselves? Sure. Is that actually what people will do in practice? No, this is well-demonstrated. People who use text generators produce more text with less sense than they would have written on their own, and this often gets worse when they're *replying* to a thread with the use of a text generator.

Note that we're not sitting behind you verifying how you're writing your text! If you can generate text and it's high enough quality that we can't tell, great, who cares, nobody will know. We *do* care if we *can* tell the text was LLM-generated, and we want to make sure there's no wiggle room in a policy that allows someone to *claim* they would have generated text like that so it's actually totally okay and not a violation. So, staking a boundary that's a little harsher than a "perfect" one would be is useful here, to make enforcing the boundary easier with less arguments.

> Any contributions or comments must be your own writing, not the product of LLMs or other tools. Do not prompt an LLM to expand on or explain your idea and then post the output; just provide the idea itself.

Author: > Perhaps we can agree on a policy that prioritizes accountability and authorship, regardless of whether a tool was used along the way.

Authorship, yes, that's the main point of policy - i.e., "your own writing" (as opposed to an LLM's writing).

But a human claiming understanding of and accountability for the LLM's outputs are not sufficient: first, because many people who read LLM output come to believe it is sensible even when it it is not, and simply being wrong about this cannot reasonably be considered a policy violation; and second, because one of the several problems this policy is aiming to address is the fact that LLM outputs can be much longer than their prompts, and it is disrespectful of reader's time to post large amounts of text generated by such a tool even if the text is technically coherent. Taking small amounts of human-written text and turning it to large amounts of LLM-written text is what many people mean by using an LLM to "enhance readability", so we can't say that using an LLM to "enhance readability" is permitted.

I don't think there is a reasonable way to draw this line other than to forbid LLM-authored text entirely, as this policy does. I acknowledge some people use LLMs to author text for a variety of reasons, and I don't have any objection to doing so when thinking through ideas yourself, but I do consider it rude to post these outputs for other people to read, and intend for the policy to capture that. I think this a widely-shared principle, even if it is not universal.

Machine translation is permissible, including translation by an LLM, but your use of translation should not introduce any new content.

> Similarly, you may use an LLM for proofreading as long as this doesn't add any new content.

Reviewer 3: I guess this is the new part, but I feel like the line of permissibility is the trickiest here—would we want to include an example of going too far? Because a misspelled word, a dropped article, or a misused preposition could all be corrected without issue, but more substantial rephrasals would quickly have authorship implications, even if the person isn't consciously thinking of it as an "addition".

> Similarly, you may use an LLM for proofreading as long as this doesn't add any new content.

Reviewer 4: ```suggestion
Similarly, you may use an LLM to proofread, simplify, condense, or improve the readability of your writing, provided that the underlying ideas and content remain entirely your own and are not substantively altered.
```

Many of us communicate effectively in our primary languages but face challenges expressing ideas clearly in English. This can lead to grammatically correct text that’s hard to follow. In such cases, basic proofreading falls short. Allowing LLMs to simplify, condense, or enhance readability (without adding new content) should be seen as an extension of proofreading that preserves the writer’s voice while improving clarity for readers.

Reviewer 2: The position expressed within plenary by many delegates was that we'd rather you post what you've written and have the reader use LLMs to do the summarisation or interpretation if that's what they desire. Anything more and we get into this gray area that you've correctly identified.
